<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dSettingsClass</class>
 <widget class="QWidget" name="dSettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mpTab">
      <attribute name="title">
       <string>Media players</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0" colspan="2">
        <widget class="QLineEdit" name="MPNamesEdit">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="enableMPCheckBox">
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="mediaPlayersLabel">
         <property name="text">
          <string>List your media player executables separated by commas (example: mpc-hc.exe):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fileTab">
      <attribute name="title">
       <string>Save to file</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QCheckBox" name="s2fCheckBox">
         <property name="text">
          <string>Save to file</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="s2fDirectoryRadio">
         <property name="text">
          <string>Select directory:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="s2fAskRadio">
         <property name="text">
          <string>Ask me every time</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="s2fDirectoryEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="s2fDirectoryButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="highlighterTab">
      <attribute name="title">
       <string>Highlighter colors</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Left button</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Right button</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="highlighterLeftButton">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="highlighterRightButton">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="highlighterLeftWidthBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>3.000000000000000</double>
           </property>
           <property name="maximum">
            <double>25.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Opacity:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="highlighterLeftOpacityLabel">
           <property name="text">
            <string>255</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="highlighterRightWidthBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>3.000000000000000</double>
           </property>
           <property name="maximum">
            <double>25.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Opacity:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="highlighterRightOpacityLabel">
           <property name="text">
            <string>255</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0" colspan="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="folderWatcherTab">
      <attribute name="title">
       <string>Folder watcher</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="enableFWCheckBox">
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="folderFWLabel">
         <property name="text">
          <string>Select folders to watch:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>400</width>
           <height>243</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="screenTab">
      <attribute name="title">
       <string>Screens</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Screenshot sources</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>If you have more than one screen select the source of screenshots</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="screen_src_primary">
            <property name="text">
             <string>Primary screen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="screen_src_cursor">
            <property name="text">
             <string>Cursor screen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="screen_src_ask">
            <property name="text">
             <string>Ask me every time when I do screenshot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="screen_src_concat">
            <property name="text">
             <string>Concatenate screenshots from all screens (like Windows does)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Area selector settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If you have more than one screen select the source of screenshots for area selector and where area selector will be opened&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="screen_as_primary">
            <property name="text">
             <string>Primary screen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="screen_as_cursor">
            <property name="text">
             <string>Cursor screen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="screen_as_ask">
            <property name="text">
             <string>Ask me every time when I do screenshot</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="settingsLabel">
     <property name="text">
      <string>Settings will be applied automatically as you click at a control.&lt;br /&gt;Settings will be deleted when the application closes.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
